qlty-cli/src/commands/plugins/disable.rs

```diff
@@ -110,11 +119,42 @@ config_version = "0"
 name = "stays"
 version = "1.0.0"
 
+[[plugin]]
+name = "to_disable"
+version = "1.0.0"
+mode = "disabled"
+
 [[plugin]]
 name = "also_stays"
 version = "1.0.0"
         "#;
 
         assert_eq!(config.document.to_string().trim(), expected.trim());
     }
+
+    #[test]
+    fn test_disable_plugin_wrong_name() {
+        let (temp_dir, _) = sample_repo();
+        let temp_path = temp_dir.path().to_path_buf();
+
+        let workspace = Workspace {
+            root: temp_path.clone(),
+        };
+        fs::create_dir_all(&temp_path.join(path_to_native_string(".qlty"))).ok();
+
+        fs::write(
```

Comment:
I assume this is repeating existing pattern but it would be nice if we can test this without touching the filesystem since it's basically a pure function (toml string in, toml string out).

If you were so inclined, it would be a nice refactor

Reply:
I don't want to hold up this PR on this particular refactor. I may look into it, but had started work on updating our `enable` subcommand and want to address that first.